interface/github.py

```diff
@@ -105,6 +105,51 @@ def org_get_teams(self):
         except GithubException as e:
             raise GithubAPIException(e.data)
 
+# ---------------------------------------------------------------
+# --------------- methods related to team members ---------------
+# ---------------------------------------------------------------
+
+    def list_team_members(self, team_id):
+        """Return a list of users in the team of id team_id."""
+        try:
+            team = self.github.get_team(team_id)
+            return map(lambda x: x, team.get_members())
+        except GithubException as e:
+            raise GithubAPIException(e.data)
+
+    def get_team_member(self, member_username, team_id):
```

Comment:
Is there a reason why this is called `member_username` but the others are called `username`?

Reply:
yes, thank you. I think it's there to indicate that the username has to be an existing team member, not just any username, otherwise, it will throw an exception. But to maintain consistency, I will change this to `username`.